src/oop/creating_objects_from_classes/hard/q3/MainTest.java

```diff
@@ -0,0 +1,102 @@
+package oop.creating_objects_from_classes.hard.q3;
+
+import global.ObjectTest;
```

Comment:
For this, as we discussed in our meeting, this category is mainly about initializing classes, (like tomato = new Tomato() )

Reply:
I will make an example of how we should do this and then send it to you